### DISCUSSION TITLE
EZ_InSAR will not start in MATLAB

### DISCUSSION THREAD

**Discussion by tamsima** [alexisInSAR/EZ-InSAR#30] [Help]
Hi again

I installed everything according to the Tutorial and would now be ready to open the EZ_InSAR GUI. However, it somehow does not work. I get the following error messages: 

Setting paths for EZ-InSAR: Matlab Interface for Easy InSAR (Version 2.0.1 Beta)...
Added to path: EZ-InSAR Version 2.0.1 Beta
>> EZ_InSAR
MathWorks::System::IUserException in WebControllerFactory::create(): [Error using matlab.internal.cef.webwindow
MATLABWindow application failed to launch. Unable to launch the MATLABWindow application. The exit code was: 127

Error in matlab.internal.webwindow/createImplementation (line 315)
                implObj = matlab.internal.cef.webwindow(varargin{:});

Error in matlab.internal.webwindow (line 163)
            obj.impl = obj.createImplementation(varargin{:});

Error in matlab.ui.internal.controller.platformhost.CEFFigurePlatformHost/createView (line 65)
                this.CEF = matlab.internal.webwindow(peerModelInfo.URL, peerModelInfo.DebugPort, position);

Error in matlab.ui.internal.controller.FigureController/createView (line 493)
                this.PlatformHost.createView(this.PeerModelInfo, pos, title, visible, resizable, windowState, windowStyle, this.Model.Uuid);

Error in matlab.ui.internal.componentframework.WebComponentController/add (line 69)
        obj.createView( parentController, parentView, propertiesStruct );

Error in matlab.ui.internal.controller.WebCanvasContainerController/add (line 43)
            add@matlab.ui.internal.componentframework.WebContainerController( this, webComponent, parentController );

Error in matlab.ui.internal.componentframework.WebControllerFactory/create (line 94)
                    component.createController( parentController, [] );

Error in matlab.ui.internal.dialog.DialogHelper.getFigureID (line 14)
                matlab.graphics.internal.drawnow.startUpdate % get ID after update

Error in matlab.ui.internal.dialog.DialogHelper.validateUIfigure (line 44)
            figureID = matlab.ui.internal.dialog.DialogHelper.getFigureID(h);

Error in matlab.ui.dialog.ProgressDialog (line 45)
            [hUIFigure, figureID] = matlab.ui.internal.dialog.DialogHelper.validateUIfigure(f);

Error in uiprogressdlg (line 33)
progressDialog = matlab.ui.dialog.ProgressDialog(hUIFigure, varargin{:});

Error in GUIMIESAR (line 35)
pg_figopen = uiprogressdlg(figopen,'Title','Loading of EZ-InSAR Application','Message','Opening the application');

Error in EZ_InSAR (line 63)
        hmain = GUIMIESAR(miesar_para);]

Not sure where I might have gone wrong. I am using Linux in a WSL for Windows. So far this was never a problem when working with MATLAB in that way. The EZ_InSAR.m and GUIMIESAR.m files are located in 
/home/mathyst/tools_insar/proc_insar/EZINSAR/EZINSAR_BIN

Thanks a lot already for your help!
Cheers
Tamara

**Comment by alexisInSAR** [comment]
I think this error is related to your installation of MATLAB and WSL. EZ-InSAR uses the App Designer to create the GUI. I have never used the WSL, so my advice could be: 
- to ask MATLAB directly
- to use a native installation of Linux
- to use a Virtual Box (Linux will be fully installed in this case). 

Sorry... 

Alexis
